Arm/BitVec.lean

```diff
@@ -289,7 +286,7 @@ abbrev lsb (x : BitVec n) (i : Nat) : BitVec 1 :=
   --   BitVec.extractLsb' i 1 x
   -- and avoid the cast here, but unfortunately, extractLsb' isn't supported
   -- by LeanSAT.
-  (BitVec.extractLsb i i x).cast (by omega)
+  (BitVec.extractLsb' i 1 x).cast (by omega)
```

Comment:
```suggestion
  BitVec.extractLsb' i 1 x
```
No need of the `cast`

Reply:
Also, remove the outdated comment from L285-288 above.

Reply:
Done.